packages/sn-filter-pane/src/components/ListboxPopoverContainer.tsx

```diff
@@ -62,6 +62,12 @@ export const ListboxPopoverContainer = ({ resources, stores }: FoldedPopoverProp
     }, transitionDuration);
   };
 
+  const closePopover = (close : boolean) => {
+    if (close) {
+      handleClose();
+    }
+  };
```

Comment:
why a close parameter? (is it ever false?)

Reply:
I agree, shall remove it.